runtime/compiler/CMakeLists.txt

```diff
@@ -115,6 +115,17 @@ if(JITSERVER_SUPPORT)
 	)
 
 	add_custom_target(j9jit_proto DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/net/gen/compile.pb.h ${CMAKE_CURRENT_SOURCE_DIR}/net/gen/compile.pb.cpp)
+
+	if(OPENSSL_BUNDLE_LIB_PATH) # --enable-openssl-bundling
+		set(OPENSSL_ROOT_DIR ${OPENSSL_BUNDLE_LIB_PATH})
+		SET(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,-rpath,$ORIGIN/..")
+	elseif(OPENSSL_DIR) # --with-openssl=fetched only
+		set(OPENSSL_ROOT_DIR ${OPENSSL_DIR})
+	endif()
+
+	include(FindOpenSSL)
+	find_package(OpenSSL REQUIRED)
+	include_directories(${OPENSSL_INCLUDE_DIR})
```

Comment:
`OPENSSL_CFLAGS` should be used somewhere here.

Reply:
`include_directories()` would have the same effect as adding `OPENSSL_CFLAGS` to `J9_CXXFLAGS` and `J9_CFLAGS`.

Reply:
We should not assume that `OPENSSL_CFLAGS` only has `-I` options.

Reply:
As per our offline discussion, I shouldn't assume `OPENSSL_CFLAGS` would contain only include path information. Will set it up in `CMAKE_ARGS` and pass it to `J9_CXXFLAGS` and `J9_CFLAGS`.

Reply:
If you want to filter out `-I` options (in the related PRs) so `OPENSSL_CFLAGS` is empty if it only had `-I` options, I can support that (as long as we have the necessary include directories via other means).

Reply:
I'll populate it to `CMAKE_ARGS` without filtering as how `CUSTOM_COMPILER_ENV_VARS` is set up to make it consistent.